tests/typescript-mock-data.spec.ts

```diff
@@ -18,6 +18,11 @@ const testSchema = buildSchema(/* GraphQL */ `
     type Query {
         user: User!
     }
+
+    input QueryInput {
```

Comment:
are you sure the name `QueryInput` makes sense? it's an input type for mutations not queries.
Maybe name it `InputType`, or like what was done with `Avatar` and `User`, try to find a name for a real life example

Reply:
Yeah, you are right
Updated